R/skip.R

```diff
@@ -20,11 +20,11 @@ skip_if_no_virtualenv <- function(python = NULL) {
   skip_if_no_python(python)
 
   key <- paste("tests", python, "virtualenv.installed", sep = ".")
-  installed <- global(key, {
+  installed <- the[[key]] %??% {
```

Comment:
Does this need to assign `the[[key]]` as well?

Reply:
Yup, good catch.